ebpf.go

```diff
@@ -35,6 +35,8 @@ type Program interface {
 	Detach() error
 	// Returns program name as it defined in C code
 	GetName() string
+	// Returns program target symbol as defined in the section name e.g. kprobe/SyS_execve
+	GetTarget() string
```

Comment:
maybe `GetNameExtras` or similar to make it kinda generic and possibly reusable for future types?

Reply:
I've changed this to be more generically applicable as `GetSection` to return the section name used for the program.